tests/foreman/cli/test_oscap.py

```diff
@@ -104,8 +104,10 @@ def setUpClass(cls):
             remote_file="/tmp/{0}".format(cls.file_name)
         )
         cls.title = 'rhel-6-content'
-        result = Scapcontent.info({'title': cls.title})
-        if not result['title'] in cls.title:
+        result = Scapcontent.list()
+        result = [scap['title'] for scap in result if
+                  scap.get('title') in cls.title]
+        if result is not cls.title:
```

Comment:
this will always pass.
You cannot compare string with string in result. You have to check for membership with in.
```
In [1]: cls.result = 'something'
In [2]: result = ['something']
In [3]: cls.result is result
Out [3]: False
In [4]: cls.result is not result
Out [3]: True

Check for membership with in
In [5]: cls.result in result
Out [5]: True
In [6]: cls.result not in result
Out [6]: False
```

Reply:
Will fix this.